e2eTest/contract_deployments_test.go

```diff
@@ -53,6 +53,68 @@ func TestContractDeployments(t *testing.T) {
 
 }
 
+func TestContractRedeployment(t *testing.T) {
+	t.Run("same contract name with different arguments", func(t *testing.T) {
+		c := newClient()
+
+		project := createProject(t, c)
+
+		contractA := `
+		pub contract HelloWorld {
+			pub var A: Int
+			pub init() { self.A = 5 }
```

Comment:
Can you include in the change init args `init()` -> `init(a: String)`

Reply:
Added function args